socorro/unittest/external/es/base.py

```diff
@@ -146,14 +132,14 @@ def get_tuned_config(self, sources, extra_values=None):
 
         return super().get_tuned_config(sources, values_source)
 
-    def get_base_config(self, es_index=None):
```

Comment:
Passing in the class to get the tuned config for here makes this more generally useful and fixes a bunch of tests that were busted.